### DISCUSSION TITLE
Share Button remap

### DISCUSSION THREAD

**Discussion by MichaelTechno** [AntiMicroX/antimicrox#604] [Ideas]
Hi
Can you add please "Share Button" to ability to remap. 
It will be great. Thanks!

**Comment by shogunreaper** [comment]
i would also like this, looking through the issues it seems like some linux users have fixed it but i can't get it working on windows 10.

**Reply by FloydNex** [reply]
same here :) Antimicro is awesome btw and with that funtionality it would be perfect
